scripts/inc.updatebinary.sh

```diff
@@ -17,11 +17,8 @@ echo '#!/sbin/ash
 # This Open GApps Shell Script Installer includes code derived from the TK GApps of @TKruzze and @osm0sis,
 # The TK GApps are available under the GPLv3 from http://forum.xda-developers.com/android/software/tk-gapps-t3116347
 #
-unzip -o "$3" '"$EXTRACTFILES"' -d /tmp;'> "$build/META-INF/com/google/android/update-binary"
```

Comment:
We use $EXTRACTFILES on purpose, because the set of files can/will change per architecture and possible new package variants.